catalog/dags/common/licenses/licenses.py

```diff
@@ -25,6 +25,12 @@ class InvalidLicenseURLException(Exception):
 
 LicenseInfo = namedtuple("LicenseInfo", ["license", "version", "url", "raw_url"])
 
+# SMK sometimes uses incorrect URL for licenses
```

Comment:
Added in here because the example SMK used in test uses this ("share-your-work") URL, and the tests fail  because of this. Also added in the `LicenseURL` PR because that's where this change actually belongs. 

I do think that we should treat the works with this URL as marked with CC0 or PDM. If others think we shouldn't, I'll update the test object to use a valid CC license URL.